packages/venia-ui/lib/components/AccountInformationPage/editForm.js

```diff
@@ -0,0 +1,115 @@
+import React, { Fragment } from 'react';
+import { func, shape, string, bool } from 'prop-types';
+
+import { mergeClasses } from '../../classify';
+import Field from '../Field';
+import TextInput from '../TextInput';
+import LinkButton from '../LinkButton';
+import Password from '../Password';
+import combine from '../../util/combineValidators';
+import {
+    hasLengthAtLeast,
+    isRequired,
+    validatePassword,
+    validateDifferentCurrentPassword
+} from '../../util/formValidators';
+import defaultClasses from './editForm.css';
+
+const EditForm = props => {
+    const {
+        classes: propClasses,
+        handleActivePassword,
+        activeChangePassword
+    } = props;
+
+    const classes = mergeClasses(defaultClasses, propClasses);
+
+    const passwordArea = !activeChangePassword ? (
+        <Fragment>
+            <div className={classes.passwordLabel}>
+                <Field id="password" label="Password">
+                    <span>{'**********'}</span>
```

Comment:
I know this was probably in the mocks but having a display at all for this "hidden" value seems odd. Personally I think it looks better without:

[![Image from Gyazo](https://i.gyazo.com/a8f3e060698c4f499f9aa8b310e6ac19.gif)](https://gyazo.com/a8f3e060698c4f499f9aa8b310e6ac19)

vs 

[![Image from Gyazo](https://i.gyazo.com/cf1d454a31f2f20b69157684a2da24ed.gif)](https://gyazo.com/cf1d454a31f2f20b69157684a2da24ed)

cc @schensley

Reply:
@schensley please confirm here

Reply:
Spoke with Scott - we can leave this as is for now. The text displayed is less helpful since the asterisks do not represent the actual password length, but it's not a big enough deal to change.

If you wanted to you could remove the `Password ******` text like I suggested, and just display the "Change Password" link, but that's up to you.

Reply:
I messed around with this - it felt weird to remove just the password text and leave the `change password` link. It also felt weird to always have the input fields displayed when you click "edit".

So I'm leaving this as is.